services/src/main/java/org/keycloak/exportimport/util/ExportUtils.java

```diff
@@ -103,15 +101,17 @@ public static RealmRepresentation exportRealm(KeycloakSession session, RealmMode
                 .map(ClientScopeModel::getName).collect(Collectors.toList()));
 
         // Clients
-        List<ClientModel> clients = Collections.emptyList();
+        List<ClientModel> clients = new ArrayList<>();
```

Comment:
Please either use `ArrayList(int initialCapacity)` or prefer `LinkedList` since the list is only added to and iterated.

Reply:
Changed it to a LinkedList as I don't want to ask a stream about its size. Pushed the code, please resolve if you're ok with it.